### DISCUSSION TITLE
Template for YX-088 aroma diffuser issue

### DISCUSSION THREAD

**Discussion by sirLeone** [arendst/Tasmota#17763] [Q&A]
### PROBLEM DESCRIPTION
After applaying template for YX-088 aroma diffuser from https://templates.blakadder.com/gernems_YX-088.html only one button is showing in web UI. From the description there should be 2 buttons: 1 for fan and 1 for led:

**Label web UI buttons:**
`Backlog WebButton1 Fan; WebButton2 Light; FriendlyName1 Diffuser Fan; FriendlyName2 Diffuser Light`

Tasmota 12.3.1

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [ ] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [ ] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [ ] Searched the problem in [discussions](https://github.com/arendst/Tasmota/discussions)
- [ ] Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
- [ ] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [ ] Device used (e.g., Sonoff Basic): _____
- [ ] Tasmota binary firmware version number used: _____
  - [ ] Pre-compiled
  - [ ] Self-compiled
- [ ] Flashing tools used: _____
- [ ] Provide the output of command: `Backlog Template; Module; GPIO 255`:
```lua
18:04:43.887 CMD: Backlog Template
18:04:43.967 MQT: stat/aromatyzerSalon/RESULT = {"NAME":"YX-088","GPIO":[0,2272,0,2304,0,0,0,0,0,0,0,0,0,0],"FLAG":0,"BASE":54}


```
- [ ] If using rules, provide the output of this command: `Backlog Rule1; Rule2; Rule3`:
```lua
18:05:22.836 CMD: Backlog Rule1
18:05:22.907 RUL: Rule1 on power1#state=0 do publish stat/%topic%/fanmode none endon on tuyareceived#dptype5id9 do publish stat/%topic%/error %value% endon  on tuyareceived#dptype4id2==0 do publish stat/%topic%/fanmode medium endon on tuyareceived#dptype4id2==1 do publish stat/%topic%/fanmode low endon on tuyareceived#dptype4id2==2 do publish stat/%topic%/fanmode high endon on tuyareceived#dptype4id3==0 do publish stat/%topic%/timer 2h timer endon on tuyareceived#dptype4id3==1 do publish stat/%topic%/timer 4h timer endon on
18:05:22.910 RUL: Rule1 +tuyareceived#dptype4id3==2 do publish stat/%topic%/timer timer off endon on tuyareceived#dptype4id6==0 do publish stat/%topic%/colormode solid endon on tuyareceived#dptype4id6==1 do publish stat/%topic%/colormode cycle endon on event#fanmode==-1 do power1 0 endon on event#fanmode!=-1 do backlog power1 1; tuyaenum1 %value% endon on event#solid do tuyaenum3 0 endon on event#cycle do tuyaenum3 1 endon
18:05:22.916 MQT: stat/aromatyzerSalon/RESULT = {"Rule1":{"State":"OFF","Once":"OFF","StopOnError":"OFF","Length":907,"Free":243,"Rules":"on power1#state=0 do publish stat/%topic%/fanmode none endon on tuyareceived#dptype5id9 do publish stat/%topic%/error %value% endon  on tuyareceived#dptype4id2==0 do publish stat/%topic%/fanmode medium endon on tuyareceived#dptype4id2==1 do publish stat/%topic%/fanmode low endon on tuyareceived#dptype4id2==2 do publish stat/%topic%/fanmode high endon on tuyareceived#dptype4id3==0 do publish stat/%topic%/timer 2h timer endon on tuyareceived#dptype4id3==1 do publish stat/%topic%/timer 4h timer endon on tuyareceived#dptype4id3==2 do publish stat/%topic%/timer timer off endon on tuyareceived#dptype4id6==0 do publish stat/%topic%/colormode solid endon on tuyareceived#dptype4id6==1 do publish stat/%topic%/colormode cycle endon on event#fanmode==-1 do power1 0 endon on event#fanmode!=-1 do backlog power1 1; tuyaenum1 %value% endon on event#solid do tuyaenum3 0 endon on event#cycle do tuyaenum3 1 endon"}}

```
- [ ] Provide the output of this command: `Status 0`:
```lua
  STATUS 0 output here:


```
- [ ] Set `weblog` to 4 and then, when you experience your issue, provide the output of the Console log:
```lua
  Console output here:


```

### TO REPRODUCE
_Steps to reproduce the behavior:_

Install fresh tasmota 12.3.1 and apply template for Gernems 400 ml Aroma Diffuser (YX-088)

### EXPECTED BEHAVIOUR
_A clear and concise description of what you expected to happen._

2 buttons in web UI : 1 for fan and 1 for led

### SCREENSHOTS
_If applicable, add screenshots to help explain your problem._

https://i.imgur.com/ESe4wL6.png

### ADDITIONAL CONTEXT
_Add any other context about the problem here._

**(Please, remember to close the issue when the problem has been addressed)**

**Comment by barbudor** [comment]
You have not provided the output of command ` status 0` 

and the output of ` Backlog Template; Module; GPIO 255`  is incomplete

**Reply by barbudor** [reply]
When doing the things correctly:

![image](https://user-images.githubusercontent.com/2996042/213880777-a358947d-9d7e-4fb8-be5f-b763569711fd.png)

**Reply by sfromis** [reply]
My guess is that you did not activate the template, as instructed in the how-to link in the left column of the templates repository page.

**Comment by sirLeone** [comment]
Both of you were right :) Somehow ctr+c and ctrl+v did not its job and after repeating pasting the template its ok. Thank you
